pytest_mergify/utils.py

```diff
@@ -0,0 +1,22 @@
+import os
+import typing
+
+CIProviderT = typing.Literal["github_actions", "circleci"]
+
+
+def get_ci_provider() -> CIProviderT | None:
+    if os.getenv("GITHUB_ACTIONS") == "true":
+        return "github_actions"
+    if os.getenv("CIRCLECI") == "true":
+        return "circleci"
+    return None
```

Comment:
shouldn't we use strtobool here also when retrieving the env value ?

Reply:
Next PR ;)